pkg/oci/spec_opts.go

```diff
@@ -1651,7 +1651,7 @@ func WithCDIDevices(devices ...string) SpecOpts {
 			return nil
 		}
 
-		registry := cdi.GetRegistry()
+		registry := cdi.GetDefaultCache()
 		if err := registry.Refresh(); err != nil {
```

Comment:
nit: you could replace this with the simpler `if err := cdi.Refresh(); err != nil {`

Reply:
Updated.